<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioCvt_Sox_gui</class>
 <widget class="QDialog" name="AudioCvt_Sox_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert Audio (SoX)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../_resources/editor.qrc">
    <normaloff>:/appicon/app.ico</normaloff>:/appicon/app.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SoX executable path:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="browse">
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="Line" name="hr1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="backup">
     <property name="text">
      <string>Make backup</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="box_resample">
     <property name="title">
      <string>Resample to</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="rate">
        <property name="text">
         <string notr="true">44100</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QProgressBar" name="progress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>332</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="5">
    <widget class="QPushButton" name="start">
     <property name="text">
      <string notr="true">Start</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>What do you want?</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" columnstretch="97,0">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="cur_section">
        <property name="text">
         <string>Convert music of current level section</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="cur_level">
        <property name="text">
         <string>Convert all music files on current level</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="cur_custom">
        <property name="text">
         <string>Convert specified files</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QListWidget" name="musics_list">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="add">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="remove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="3" rowspan="2" colspan="3">
    <widget class="QTabWidget" name="convertoTo">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="dont_change">
      <attribute name="title">
       <string>Don't change format</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>The files' format will not be changed</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="to_mp3">
      <attribute name="title">
       <string notr="true">MP3</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QGroupBox" name="mp3_set_birtate">
         <property name="minimumSize">
          <size>
           <width>89</width>
           <height>53</height>
          </size>
         </property>
         <property name="title">
          <string>Set bitrate</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QComboBox" name="mp3_bitrate">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>13</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">40</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">48</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">56</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">64</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">80</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">96</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">112</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">128</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">160</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">192</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">224</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">256</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">320</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Files will be converted into MP3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>219</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="to_flac">
      <attribute name="title">
       <string notr="true">FLAC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Files will be converted into FLAC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="to_ogg">
      <attribute name="title">
       <string notr="true">OGG</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Files will be converted into OGG</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ogg_set_quality">
         <property name="minimumSize">
          <size>
           <width>317</width>
           <height>50</height>
          </size>
         </property>
         <property name="title">
          <string>Set quality</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QSlider" name="ogg_quality">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ogg_quality_var">
            <property name="text">
             <string notr="true">5</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="to_wav">
      <attribute name="title">
       <string notr="true">WAV</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Files will be converted into WAV</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Here you can quickly convert your music or SFX files which have been&lt;br&gt;formatted incorrectly (wrong sample rate, bad codec, etc.).&lt;/p&gt;
&lt;p&gt;This feature uses the &lt;a href=&quot;http://sox.sourceforge.net/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;SoX&lt;/span&gt;&lt;/a&gt; audio converter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="sox_bin_path"/>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>102</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Convert into new format
(select a tab to choose target format)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../_resources/editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mp3_set_birtate</sender>
   <signal>toggled(bool)</signal>
   <receiver>mp3_bitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_resample</sender>
   <signal>toggled(bool)</signal>
   <receiver>rate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cur_custom</sender>
   <signal>toggled(bool)</signal>
   <receiver>musics_list</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cur_custom</sender>
   <signal>toggled(bool)</signal>
   <receiver>add</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cur_custom</sender>
   <signal>toggled(bool)</signal>
   <receiver>remove</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
